cypress/pages/slices/slicePage.js

```diff
@@ -49,4 +49,41 @@ export class SlicePage {
     cy.contains("Simulate Slice").click();
     return this;
   }
+
+  /**
+   * add a new variation through the variation modal.
+   *
+   * @param {string} variationName Name of the variation.
+   */
+  addVariation(variationName) {
+    cy.get("[aria-label='Expand variations']").click({ force: true });
+    cy.contains("button", "Add new variation").click();
+
+    cy.get("[aria-modal]").within(() => {
+      cy.getInputByLabel("Variation name*").type(variationName);
+      cy.contains("button", "Submit").click();
+    });
+    cy.get("[aria-modal]").should("not.exist");
+
+    return this;
+  }
+
+  /**
+   * add a new widget through the modal form.
+   *
+   * @param {string} name Name of the widget field.
+   * @param {string} type Type of the widget.
+   */
+  addNewWidgetField(name, type) {
```

Comment:
is this different from `cypress/helpers/slice` ?

Reply:
It replaces it